src/components/PlatformOnboardingGetStarted/PlatformOnboardingGetStarted.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { useContext } from 'react'
+import { PlatformOnboardingContext } from '../../contexts/PlatformOnboardingContext'
+import { PlatformOnboardingViewProps } from '../../views/PlatformOnboardingView/PlatformOnboardingView'
+import { Button } from '../Button'
+import { Heading, Text, TextSize } from '../Typography'
+
+export const PlatformOnboardingGetStarted = ({
+  stringOverrides,
+}: PlatformOnboardingViewProps) => {
+  const { nextStep } = useContext(PlatformOnboardingContext)
+  return (
+    <div className='Layer__platform__onboarding__header'>
+      <Heading>Welcome to {stringOverrides?.header}</Heading>
+      <Text as='p' size={TextSize.md}>
```

Comment:
NIT: `Text` defaults `as` to `'p'`, so this could be skipped. 
```suggestion
      <Text size={TextSize.md}>
```